spec/role_model_spec.rb

```diff
@@ -181,37 +181,38 @@
       it "should add a role given as a symbol" do
         subject.roles << :third
         subject.roles.should include(:foo, :bar, :third)
-        subject.should have(3).roles
+        expect(subject.roles.size).to eq(3)
       end
 
       it "should add a role given as a string" do
         subject.roles << 'third'
         subject.roles.should include(:foo, :bar, :third)
-        subject.should have(3).roles
+        expect(subject.roles.size).to eq(3)
       end
 
       it "should not show a role twice in the return value" do
-        (subject.roles << :foo).should have(2).roles
+        subject.roles << :foo
+        expect(subject.roles.size).to be 2
```

Comment:
Since all the other examples say `.to eq(n)` it would be nice to use the same form here as well.